### PR TITLE
Full upgrade

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- *Full upgrade*

## Solution
- [x] *Apply example_ynh*
- [x] *Fix package_check errors*
- [x] *Upgrade to 0.18.3*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
- [x] **Code review**
- [x] **Approval (LGTM)**  
*Code review and approval have to be from a member of @YunoHost/apps group*
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/funkwhale_ynh_PR67%20(yalh76)/)

Revision: 42a905db55457d01cb8884d7e9ebfa2178d97f2f 
Capture

When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by yalh76** [comment]
PR #67 ready

File: check_process [hunk 1]
```diff
@@ -16,8 +16,6 @@
 		setup_private=1
 		setup_public=1
 		upgrade=1
-		upgrade=1		from_commit=80a5044d13a6782063fee2d7fb7a01832a6aa767
-		upgrade=1               from_commit=806bcb0f320d8e3e28711a9ae1d2b175b95c65ce
```
Comment (Josue-T) [inline check_process]: Why, do you remove the check of the upgrade from the old version ?

Comment (yalh76) [reply check_process]: Because those versions don't install anymore

Comment (Josue-T) [reply check_process]: Ok it still interesting to know if the migration work...

I my side, what I do (by example in synapse) is to make a specific branch based on this commit named by example "Old_version_for_CI_1". And I just add a patch to fix the install of this old version.

Comment (yalh76) [reply check_process]: OK, then, I let funkwhame_ynh maintainer manage that :)

**Pull request merged**